GuessTheNumber/GuessTheNumber/SecondViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  SecondViewController.swift
+//  GuessTheNumber
+//
+//  Created by Nick Krasnitskiy on 26.06.2021.
+//
+
+import UIKit
+
+class SecondViewController: UIViewController {
+    
+    var delegate: ViewControllerDelegate?
```

Comment:
Delegates should be weak to avoid reference cycles ([Weak Delegate](https://realm.github.io/SwiftLint/weak_delegate.html)).

Reply:
Thanks. Fixed.